helm/alfresco-content-services/values.yaml

```diff
@@ -53,13 +53,33 @@ apiexplorer:
   ingress:
     path: /api-explorer
 
+transformrouter:
+  replicaCount: 1
+  image:
+    repository: quay.io/alfresco/alfresco-transform-router
+    tag: "0.0.11-EA"
+    pullPolicy: Always
+    internalPort: 8095
+  service:
+    name: transform-router
+    type: ClusterIP
+    externalPort: 80
+  readinessProbe:
+    initialDelaySeconds: 140
+    periodSeconds: 60
+    timeoutSeconds: 10
+  livenessProbe:
+    initialDelaySeconds: 140
+    periodSeconds: 120
+    timeoutSeconds: 60
+
 # Declares the alfresco-pdf-renderer service used by the content repository
 # to transform pdf files
 pdfrenderer:
   replicaCount: 1
   image:
     repository: alfresco/alfresco-pdf-renderer
```

Comment:
Management are expecting that the updated images remain in quay.io.
The 1.N line can stay in Docker Hub as they are used directly by the current trial, which does not required quay.io access.

Reply:
Ok, we'll remove transformers 2.0.0-RC1, 2.0.0-RC2 and latest from Docker Hub.

Reply:
@pnirwal @derekhulley Hmm ... my understanding (not being present at "the meeting") was that this applies to Transform Router image only ? 

Otherwise, we end up with fork of maintenance process (build & release plan) for T-Engines (if we ever need to back-port hot-fixes from 2.N to 1.N).

Reply:
That is, unfortunately, a detail.
The box drawn with JN yesterday went around all the images.

Reply:
The decision as to whether "scale" is protected is what is important.

Reply:
OK. We're tracking via ATS-122. I've added the "detail" consequence as a [comment](https://issues.alfresco.com/jira/browse/ATS-122?focusedCommentId=584605&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-584605).